Olha_Lesiuk/task_1.py

```diff
@@ -0,0 +1,15 @@
+list_of_elements = [1, 2, 3, 4, 5, 6, 7, 8]
+Oddlist = []
+EvenList = []
+
+for i in list_of_elements:
```

Comment:
try using more understandable varieble names in future. except i use number in such case